docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md

```diff
@@ -16,110 +16,59 @@ ms.assetid: fab89cb8-dac1-4e21-9d34-5eadd5213c05
 
 # Create an Azure service principal with Azure CLI 2.0
 
```

Comment:
Putting this up at the top-what I was looking for was for the content to be more prescriptive. More step by step, with the core scenario being to use the Azure CLI to create a service principal and then use the service principal (instead of the default service prinicpal created when you do the device login) to log in to the Azure CLI and run some commands.